ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewHeaderFooterViewDemoController.swift

```diff
@@ -33,6 +33,7 @@ class TableViewHeaderFooterViewDemoController: DemoController {
         container.spacing = 0
 
         navigationController?.navigationBar.shadowImage = UIImage()
+        navigationController?.navigationBar.backgroundColor = Colors.navigationBarBackground
```

Comment:
Can we use a color token instead?

Reply:
This is in main, so using Colors.swift is in line with the rest of the implementations. This will be updated eventually along with the removal of Colors.swift in the fluent2-colors branch.